translations/wesnoth-The_Earths_Gut/ja.po

```diff
@@ -9097,8 +9023,10 @@ msgstr "モンスターどもめ、毒をもった奴らをここに送り込ん
 msgid "Ooh... warn the queen, the rebels are getting tough..."
 msgstr "おぉ...女王様に警告してくれ、反乱軍が強くなっていく..."
 
-# [ja personal memo] bug repo (前に記述したものと同じ内容): unit が下船した状態を save して reload すると、下船したユニットが消えている。乗船直後に下船しても同じ状況になる。
-# [ja] bug report (same as the previous one):When a unit disembarked and saved, and then reloaded, the disembarked unit is gone.
+# [ja personal memo] bug repo (前に記述したものと同じ内容 see ID962 ): unit が下船した状態を save して reload すると、下船したユニットが消えている。乗船直後に下船しても同じ状況になる。
+# [ja] bug report (same as the previous one):
+# When a unit disembarked and saved, and reloaded
+#  -> the disembarked unit is gone.
```

Comment:
I think this is issue #27, for reference

Reply:
Yes, I think you are right.
Sorry, I had left some things I had written before as TODOs for myself.
 I had also checked #27 earlier. I even had a rough look at the source files, but I wasn't skilled enough to fix it properly.